postgres0.yml

```diff
@@ -46,6 +46,7 @@ postgresql:
     env_dir: /home/postgres/etc/wal-e.d/env
     threshold_megabytes: 10240
     threshold_backup_size_percentage: 30
+  restore: /usr/bin/true
```

Comment:
We should not reference anything with absolute path...
Because for example in Ubuntu `true` is located just in `/bin`...